src/server/server_family.h

```diff
@@ -162,6 +162,12 @@ class ServerFamily {
   bool AwaitDispatches(absl::Duration timeout,
                        const std::function<bool(util::Connection*)>& filter);
 
+  // Sets the server to replicate another instance. Does not flush the database beforehand!
+  // Sets 'success' to true/false depending on whether the replication connected successfully or
+  // not. If 'success' is null, it is ignored.
+  // ! Note: the method does not modify 'host', 'port' but requires them to be references.
```

Comment:
Why not pass `const&` then?

Reply:
This is because `CmdArgList` is a span of `absl::Span<char>`, which cannot be constructed from a `const string&`.

Very annoying, even more so since `ReplicaOf` doesn't modify its arguments at all, but that's how it is.

Reply:
I think that a better design is to have `ReplicaInternal` accept 2 `string_view`s (host and ip), and call it with the values when you have the flag, or parse the args and pass them as string views when you have `CmdArgList`

i.e. move the lines
```cpp
  std::string_view host = ArgS(args, 0);
  std::string_view port_s = ArgS(args, 1);
```
To outside `ReplicaInternal()` and these would be the arguments passed to it

Reply:
Should have thought of this earlier 🤦